ext/opentelemetry-ext-requests/src/opentelemetry/ext/requests/__init__.py

```diff
@@ -92,8 +92,9 @@ def instrumented_request(self, method, url, *args, **kwargs):
             span.set_attribute("http.method", method.upper())
             span.set_attribute("http.url", url)
 
-            headers = kwargs.setdefault("headers", {})
+            headers = kwargs.get("headers", {}) or {}
```

Comment:
Any thoughts on checking if headers if `None` here and just skipping the call to `inject` if it's the case?